[2025-02-06T15:29:21.714+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-06T15:29:21.732+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_dag_incremental.extract_online_store_data scheduled__2025-02-06T00:02:00+00:00 [queued]>
[2025-02-06T15:29:21.742+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_dag_incremental.extract_online_store_data scheduled__2025-02-06T00:02:00+00:00 [queued]>
[2025-02-06T15:29:21.743+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-02-06T15:29:21.758+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): extract_online_store_data> on 2025-02-06 00:02:00+00:00
[2025-02-06T15:29:21.770+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=355) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-06T15:29:21.771+0000] {standard_task_runner.py:72} INFO - Started process 361 to run task
[2025-02-06T15:29:21.772+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'etl_dag_incremental', 'extract_online_store_data', 'scheduled__2025-02-06T00:02:00+00:00', '--job-id', '320', '--raw', '--subdir', 'DAGS_FOLDER/etl.py', '--cfg-path', '/tmp/tmpa0cx0tyy']
[2025-02-06T15:29:21.773+0000] {standard_task_runner.py:105} INFO - Job 320: Subtask extract_online_store_data
[2025-02-06T15:29:21.823+0000] {task_command.py:467} INFO - Running <TaskInstance: etl_dag_incremental.extract_online_store_data scheduled__2025-02-06T00:02:00+00:00 [running]> on host f34e19f1f8c5
[2025-02-06T15:29:21.925+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_dag_incremental' AIRFLOW_CTX_TASK_ID='extract_online_store_data' AIRFLOW_CTX_EXECUTION_DATE='2025-02-06T00:02:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-06T00:02:00+00:00'
[2025-02-06T15:29:21.926+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-02-06T15:29:21.926+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-02-06T15:29:21.926+0000] {logging_mixin.py:190} INFO - Current task name:extract_online_store_data state:running start_date:2025-02-06 15:29:21.732395+00:00
[2025-02-06T15:29:21.927+0000] {logging_mixin.py:190} INFO - Dag name:etl_dag_incremental and current dag run status:running
[2025-02-06T15:29:21.927+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-06T15:29:21.951+0000] {base.py:84} INFO - Retrieving connection 'online_store_conn'
[2025-02-06T15:29:21.956+0000] {sql.py:553} INFO - Running statement: 
            SELECT * FROM online_orders 
            WHERE created_at > '2000-01-01 00:00:00'
            ORDER BY created_at ASC;
        , parameters: None
[2025-02-06T15:29:21.968+0000] {sql.py:562} INFO - Rows affected: 5155
[2025-02-06T15:29:22.106+0000] {etl.py:150} INFO - Extracted orders 5155 new rows from db_online_store.
[2025-02-06T15:29:22.109+0000] {sql.py:553} INFO - Running statement: 
            SELECT * FROM customers 
            WHERE updated_at > '2000-01-01 00:00:00'
            ORDER BY created_at ASC;
        , parameters: None
[2025-02-06T15:29:22.113+0000] {sql.py:562} INFO - Rows affected: 3016
[2025-02-06T15:29:22.148+0000] {etl.py:170} INFO - Extracted customers 3016 new rows from db_online_store.
[2025-02-06T15:29:22.151+0000] {python.py:240} INFO - Done. Returned value was: ('/tmp/online_oders_data.json', '/tmp/online_customer_data.json')
[2025-02-06T15:29:22.171+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-06T15:29:22.171+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=etl_dag_incremental, task_id=extract_online_store_data, run_id=scheduled__2025-02-06T00:02:00+00:00, execution_date=20250206T000200, start_date=20250206T152921, end_date=20250206T152922
[2025-02-06T15:29:22.180+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-02-06T15:29:22.181+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-02-06T15:29:22.181+0000] {logging_mixin.py:190} INFO - Dag name:etl_dag_incremental queued_at:2025-02-06 15:29:17.219597+00:00
[2025-02-06T15:29:22.181+0000] {logging_mixin.py:190} INFO - Task hostname:f34e19f1f8c5 operator:PythonOperator
[2025-02-06T15:29:22.226+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-06T15:29:22.253+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-06T15:29:22.254+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-02-07T11:22:12.973+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-07T11:22:12.998+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_dag_incremental.extract_online_store_data scheduled__2025-02-06T00:02:00+00:00 [queued]>
[2025-02-07T11:22:13.010+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_dag_incremental.extract_online_store_data scheduled__2025-02-06T00:02:00+00:00 [queued]>
[2025-02-07T11:22:13.010+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-02-07T11:22:13.040+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): extract_online_store_data> on 2025-02-06 00:02:00+00:00
[2025-02-07T11:22:13.063+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=129) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-07T11:22:13.067+0000] {standard_task_runner.py:72} INFO - Started process 134 to run task
[2025-02-07T11:22:13.068+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'etl_dag_incremental', 'extract_online_store_data', 'scheduled__2025-02-06T00:02:00+00:00', '--job-id', '617', '--raw', '--subdir', 'DAGS_FOLDER/etl.py', '--cfg-path', '/tmp/tmpdeuxhmx2']
[2025-02-07T11:22:13.071+0000] {standard_task_runner.py:105} INFO - Job 617: Subtask extract_online_store_data
[2025-02-07T11:22:13.154+0000] {task_command.py:467} INFO - Running <TaskInstance: etl_dag_incremental.extract_online_store_data scheduled__2025-02-06T00:02:00+00:00 [running]> on host f34e19f1f8c5
[2025-02-07T11:22:13.336+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_dag_incremental' AIRFLOW_CTX_TASK_ID='extract_online_store_data' AIRFLOW_CTX_EXECUTION_DATE='2025-02-06T00:02:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-06T00:02:00+00:00'
[2025-02-07T11:22:13.338+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-02-07T11:22:13.338+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-02-07T11:22:13.339+0000] {logging_mixin.py:190} INFO - Current task name:extract_online_store_data state:running start_date:2025-02-07 11:22:12.998606+00:00
[2025-02-07T11:22:13.339+0000] {logging_mixin.py:190} INFO - Dag name:etl_dag_incremental and current dag run status:running
[2025-02-07T11:22:13.340+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-07T11:22:13.370+0000] {base.py:84} INFO - Retrieving connection 'online_store_conn'
[2025-02-07T11:22:13.375+0000] {sql.py:553} INFO - Running statement: 
            SELECT * FROM online_orders 
            WHERE created_at > '2000-01-01 00:00:00'
            ORDER BY created_at ASC;
        , parameters: None
[2025-02-07T11:22:13.411+0000] {sql.py:562} INFO - Rows affected: 9492
[2025-02-07T11:22:13.775+0000] {etl.py:150} INFO - Extracted orders 9492 new rows from db_online_store.
[2025-02-07T11:22:13.780+0000] {sql.py:553} INFO - Running statement: 
            SELECT * FROM customers 
            WHERE updated_at > '2000-01-01 00:00:00'
            ORDER BY created_at ASC;
        , parameters: None
[2025-02-07T11:22:13.789+0000] {sql.py:562} INFO - Rows affected: 3603
[2025-02-07T11:22:13.861+0000] {etl.py:170} INFO - Extracted customers 3603 new rows from db_online_store.
[2025-02-07T11:22:13.874+0000] {python.py:240} INFO - Done. Returned value was: ('/tmp/online_oders_data.json', '/tmp/online_customer_data.json')
[2025-02-07T11:22:13.907+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-07T11:22:13.907+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=etl_dag_incremental, task_id=extract_online_store_data, run_id=scheduled__2025-02-06T00:02:00+00:00, execution_date=20250206T000200, start_date=20250207T112212, end_date=20250207T112213
[2025-02-07T11:22:13.922+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-02-07T11:22:13.923+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-02-07T11:22:13.923+0000] {logging_mixin.py:190} INFO - Dag name:etl_dag_incremental queued_at:2025-02-07 11:22:04.748334+00:00
[2025-02-07T11:22:13.923+0000] {logging_mixin.py:190} INFO - Task hostname:f34e19f1f8c5 operator:PythonOperator
[2025-02-07T11:22:13.974+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-07T11:22:14.018+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-07T11:22:14.021+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
